specifyweb/statistics/views.py

```diff
@@ -0,0 +1,126 @@
+                                            'type': 'integer'
+                                        }
+                                    }
+                                },
+                                'preparation': {
+                                    'type': 'object',
+                                    'additionalProperties': True,
+                                },
+                                'typeSpecimen': {
+                                    'type': 'object',
+                                    'additionalProperties': True,
+                                },
+                                'localityGeography': {
+                                    'type': 'object',
+                                    'properties': {
+                                        'countries': {
+                                            'type': 'integer'
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                },
+            }
+        }
+    }}, )
+def collection_global(request) -> http.HttpResponse:
+    cursor = connection.cursor()
+    # Holdings
+    # Families
+    holding_dict = {}
+    cursor.execute("""
+    SELECT count(*) FROM (SELECT DISTINCT tx.TaxonID FROM (SELECT DISTINCT tax.TaxonID,tax.nodenumber FROM (SELECT TaxonID FROM determination WHERE CollectionMemberID = %s AND determination.IsCurrent = true) as Families1,taxon as tax WHERE tax.isaccepted <> 0 and tax.TaxonID = Families1.TaxonID) as Families2, taxon as tx WHERE tx.rankid = 140 and Families2.nodenumber between tx.nodenumber and tx.HighestChildNodeNumber) as Familes3
+                  """, [request.specify_collection.id])
+    holding_dict['familiesRepresented'] = int(cursor.fetchone()[0])
+
+    # Genera represented
+    cursor.execute("""
+       SELECT count(DISTINCT tx.TaxonID) FROM (SELECT DISTINCT tax.TaxonID,tax.nodenumber FROM (SELECT TaxonID FROM determination WHERE CollectionMemberID = %s AND determination.IsCurrent <> 0) as Genera1,taxon as tax WHERE tax.isaccepted <> 0 and tax.TaxonID = Genera1.TaxonID) as Genera2, taxon as tx WHERE tx.rankid = 180 and Genera2.nodenumber between tx.nodenumber and tx.HighestChildNodeNumber""",
+                   [request.specify_collection.id])
+    holding_dict['generaRepresented'] = int(cursor.fetchone()[0])
+
+    # UNQ_SPEC
+    cursor.execute("""
+       SELECT count(DISTINCT tx.TaxonID) FROM (SELECT DISTINCT tax.TaxonID,tax.nodenumber FROM (SELECT TaxonID FROM determination WHERE CollectionMemberID = %s AND determination.IsCurrent <> 0) as Genera1,taxon as tax WHERE tax.isaccepted <> 0 and tax.TaxonID = Genera1.TaxonID) as Genera2, taxon as tx WHERE tx.rankid = 220 and Genera2.nodenumber between tx.nodenumber and tx.HighestChildNodeNumber"""
+                   , [request.specify_collection.id])

```

Comment:
Please format all of these consistently (i.e, comma should be at the end of the line, not at the beginning)